cmd/admin-info-server.go

```diff
@@ -327,7 +327,7 @@ func mainAdminServerInfo(ctx *cli.Context) error {
 		storageInfoStat := backendStatus{}
 
 		for index, mountPath := range storageInfo.MountPaths {
-			if strings.Contains(mountPath, serverInfo.Addr) || serverInfo.Addr[0] == '/' || serverInfo.Addr[0] == '.' {
+			if strings.Contains(mountPath, serverInfo.Addr) || mountPath[0] == '/' || serverInfo.Addr[0] == '/' || serverInfo.Addr[0] == '.' {
```

Comment:
can these type of functions be converted to some `ifThis` style function? to avoid reading this laborious line?

Reply:
ok